DHCPServer/Library.Tests/UnitTest1.cs

```diff
@@ -0,0 +1,27 @@
+using CodePlex.JPMikkers.DHCP;
+using DHCPServerApp;
+using NUnit.Framework;
+using System.Linq;
+using System.Threading;
+
+namespace Library.Tests
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void Test1()
```

Comment:
This test would not work in an automated build environment, can you comment it out for now?